moveit_kinematics/cached_ik_kinematics_plugin/include/moveit/cached_ik_kinematics_plugin/cached_ik_kinematics_plugin.h

```diff
@@ -177,27 +177,27 @@ class IKCacheMap : public std::unordered_map<std::string, IKCache*>
 };
 
 // Helper class to enable/disable initialize() methods with new/old API
-// hasRobotModelAPI<T>::value provides a true/false constexpr depending on KinematicsPlugin offers the new API
+// HasRobotModelAPI<T>::value provides a true/false constexpr depending on KinematicsPlugin offers the new API
 // This uses SFINAE magic: https://jguegant.github.io/blogs/tech/sfinae-introduction.html
 template <typename KinematicsPlugin, typename = bool>
-struct hasRobotModelAPI : std::false_type
+struct HasRobotModelAPI : std::false_type  // NOLINT(readability-identifier-naming)
```

Comment:
Why are we not following our own name formatting rules here?

Reply:
Do you prefer `HasRobotModelApi`? For me, API is always UPPERCASE only.

Reply:
I would prefer `HasRobotModelApi` unless we are consistently making API capitalized in the whole codebase and we add this as a rule in our style guide.

Reply:
Done.